js/constants.js

```diff
@@ -0,0 +1,19 @@
+//create button
+export const createTask = document.getElementById("createButton");
+
+//content  task list
+export const content = document.querySelector(".content");
+export const list = document.getElementById("list");
+
+//Input BOX: add to task input box, add task button, trash button
+export const inputTask = document.getElementById("inputTask");
+export const input = document.getElementById("input");
+export const addTask = document.getElementById("addTaskButton");
+export const trashInput = document.querySelector(".trashInput");
```

Comment:
What does transInput denote here?

Reply:
trash = 
![img_464783](https://user-images.githubusercontent.com/129286746/230087076-a274014b-652d-4497-bf26-41a44ef1e842.png)

Reply:
trashInput resembles here trash icon

Reply:
Then can we go for a bit more specific naming? For the classname/ id we can go for `deleteButton` and for the const we can go for `DELETE_TASK`